### DISCUSSION TITLE
Does Parcel not support live/hot reload for HTML and CSS only?

### DISCUSSION THREAD

**Discussion by titungdup** [parcel-bundler/parcel#7290] [Q&A] [answered]
I am trying to build a static page with HTML, CSS(SASS, Tailwind), and a little bit of JS. But the live reload is not working for both HTML and CSS. Is Parcel not capable of doing it?

**Comment by titungdup** [comment, marked as answer]
Got it working :)

**Reply by jaitaiwan** [reply]
Could you say how you got it working?

**Reply by BryanCanCode313** [reply]
yes what did you do?

**Reply by titungdup** [reply]
I got it working by adding ```type="module"``` on script tag. Before i was using ```type="text/javascript"``` not sure why it didn't worked. @jaitaiwan @BryanCanCode313

**Reply by kccodemonkey** [reply]
Thanks for the tip! It pointed me to this [article](https://parceljs.org/blog/rc0/) which explains why 'type=module' is required in Parcel 2.
